react/src/lib/components/DeckGLMap/components/DeckGLWrapper.tsx

```diff
@@ -18,7 +18,7 @@ import { DrawingLayer, WellsLayer, PieChartLayer } from "../layers";
 import { Layer, View } from "deck.gl";
 import { DeckGLView } from "./DeckGLView";
 import { Viewport } from "@deck.gl/core";
-import { colorTablesArray } from "./ColorTableTypes";
```

Comment:
Can this file be removed now?

Reply:
Yes, will test once and submit changes.

Reply:
Since this file is used inside discrete and continuous legend currently, it is better to remove later once we use legend from color table repo.